packages/mjml-parser-xml/src/index.js

```diff
@@ -172,25 +186,41 @@ export default function MJMLParser(xml, options = {}, includedIn = []) {
 
         cur = newNode
       },
-      onclosetag: () => {
+      onclosetag: name => {
+
+        if (endingTags.indexOf(name) !== -1) {
+          inEndingTag -= 1
+
+          if (!inEndingTag) { // we're getting out of endingTag
```

Comment:
Need to refactor this a bit : 
- create small helper function like `isSelfClosing` 
- use the `lodash` find function instead of indexOf to harmonize everything.

Did you test with mj-include ?

Reply:
yup added new test case with mj-include